CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Change Log
 ## 0.7.0 (Sep 11th 2023)
-- build - add zgrinber as jenkins release approver [#629](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/629)
-- build - add ishishov as jenkins release approver  [#634](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/634)
-- build - add vbelouso as release approver [#635](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/635)
-- refactor - crda to rhda alpha [#636](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/636)
+- fixes - Improved overall performance and stability with the analysis report.
+- informational - Alpha release of the new Red Hat Dependency Analytics (RHDA) extension.
+- informational - The deprecated CRDA GitHub Action CI tool has been disabled.
```

Comment:
Please remove this line. GitHub action has no relevance to users of our VSCode extension

Reply:
Thanks for the feedback!  I'll remove it.